### PR TITLE
Remove symlinks and use an editable package with pipenv

### PR DESCRIPTION
**Pull request by ryanwersal**

### CODE REVIEW

File: libsabergen/setup.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import setuptools
+
+setuptools.setup(
+    name="libsabergen",
+    version="0.0.1",
+    author="beatsabertools Contributors",
+    author_email="",
+    description="libsabergen description",
+    packages=setuptools.find_packages(),
+)
```
Comment (zvekovius) [inline libsabergen/setup.py]: newline.

**Comment by zvekovius** [comment]
I'll test on my environment today.

File: libsabergen/setup.py (hunk 1 above)
Comment (ryanwersal) [reply libsabergen/setup.py]: Fixed 👍

Comment (zvekovius) [reply libsabergen/setup.py]: :D

**Comment by zvekovius** [comment]
> I'll test on my environment today.

I lied. I broke things getting python3.7 on my box.

**Comment by zvekovius** [comment]
fwiw. lgtm :) worked on my env after building some python from source.

**Comment by ryanwersal** [comment]
I tested it on Windows and macOS and ensured the DevOps builds worked before merging 🗡

**Pull request merged**